templates/flavors/clusterclass/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+namespace: default
+resources:
+  - patches/clusterclass.yaml
+  - patches/cluster.yaml
```

Comment:
I like how you split up each resource here, makes the templates a lot more modular. I think we should consider doing this for more of our templates. 

consider taking the base resources out of the "patches" directory since they aren't patches but base resources.